src/frontend/src/pages/ChangeRequestDetailPage/ReviewChangeRequestView.tsx

```diff
@@ -130,7 +150,7 @@ const ReviewChangeRequestsView: React.FC<ReviewChangeRequestViewProps> = ({
               );
             })}
           </Box>
-          <form id={'review-notes-form'} onSubmit={handleSubmit(onSubmitWrapper)}>
+          <form id={'review-notes-form'} onSubmit={handleSubmit(handleShowWarning)}>
```

Comment:
just combine the handle show warning into the onSubmitWrapper because onSubmitWrapper already does nothing anyways

Reply:
sometimes i want to just do onsubmit wrapper and other times i dont